src/mixins/filing-template-mixin.ts

```diff
@@ -389,12 +389,24 @@ export default class FilingTemplateMixin extends DateMixin {
           identifier: this.getBusinessId,
           legalType: this.getEntityType
         },
-        parties: this.getOrgPeople,
+        parties: null, // applied below
         offices: this.getOfficeAddresses,
         contactPoint: this.getContactPoint
       }
     }
+    // Apply parties to filing
+    {
+      // make a copy so we don't change original array
+      let parties = cloneDeep(this.getOrgPeople)
+
+      // prepare parties
+      parties = isDraft ? parties : this.prepareParties(parties)
+
+      // fix schema issues
+      parties = this.fixPartySchemaIssues(parties)
 
```

Comment:
This also fixed the bug that New added person and removed person are not updated - ticket
https://app.zenhub.com/workspaces/entities---olga-65af15f59e89f5043c2911f7/issues/gh/bcgov/entity/23277
This ticket can be verified and done now

Reply:
If your current solution passes all the tests then you can go and add a comment in the above ticket and resolve it.